Makefile

```diff
@@ -329,6 +332,14 @@ playbook-mock-image: build/playbook-mock/Dockerfile $(BINDIR)/playbook-mock
 	docker tag $(PLAYBOOK_MOCK_IMAGE) $(REGISTRY)playbook-mock:$(VERSION)
 	docker tag $(PLAYBOOK_MOCK_MUTABLE_IMAGE) $(REGISTRY)playbook-mock:$(MUTABLE_TAG)
 
+.PHONY: $(CLUSTERAPI_BIN)/apiserver
+$(CLUSTERAPI_BIN)/apiserver: .apiServerBuilderImage
+	mkdir -p $(PWD)/$(CLUSTERAPI_BIN) && docker run -i -v $(PWD)/$(CLUSTERAPI_BIN):/output --entrypoint=/bin/bash apiserverbuilderimage -c "export GOPATH=/go && mkdir -p /go/src/k8s.io/kube-deploy && cd /go/src/k8s.io/kube-deploy && git clone https://github.com/kubernetes/kube-deploy.git . && cd cluster-api && apiserver-boot build executables --generate=false && cp bin/* /output"
+
+.PHONY: cluster-api-image
+cluster-api-image: $(CLUSTERAPI_BIN)/apiserver build/clusterapi-image/Dockerfile
+	cp build/clusterapi-image/Dockerfile $(CLUSTERAPI_BIN) 
+	docker build -t clusterapi ./$(CLUSTERAPI_BIN)
```

Comment:
Suggest "cluster-api" to match the make target a little better. Might not be a terrible idea to use "kubernetes-cluster-api" as well.